torch/ao/quantization/qconfig.py

```diff
@@ -94,13 +97,68 @@ class QConfig(namedtuple('QConfig', ['activation', 'weight'])):
           activation=MinMaxObserver.with_args(dtype=torch.qint8),
           weight=default_observer.with_args(dtype=torch.qint8))
 
+
+    **QConfig v1**
+    QConfig v1 (current QConfig) was introduced when eager mode quantization is designed,
+    it has two keys: "activation" and "weight", here
+    activation means the output activation, and weight means weight attributes for weighted modules
+
+    Later we developed torchscript graph mode quantization (quantize_jit) and fx graph
+    mode quantization (quantize_fx), and now quantization in pytorch 2.0 export path
+    (quantize_pt2e), but we did not redesign QConfig, and "activation" is redefined as a
+    setting for both input and output, "weight" also expanded it's definition serving both setting for module\
+    weigt and the weight input of torch functional or torch ops (with the aid of extra
+    configurations to specify which input is weight) and we do not have a way to configure
+    the observer/fake_quant constructor for bias.
+
+    **QConfig v2**
+    This PR adds keys for the second version of QConfig, it has the following keys:
+    ["input_args", "module_attrs", "output", "version"]
+
+    "input_args": is used to configure the observer/fakequant constructors for input arguments,
```

Comment:
So for the `QConfig v2`, the weight input of torch functional will be set here instead of the `module_attrs`.

Reply:
it should be a part of `input_args`, we can add `input_kwargs` later if needed